subprojects/core/src/testFixtures/groovy/org/gradle/process/ShellScript.groovy

```diff
@@ -178,7 +178,7 @@ abstract class ShellScript {
 
         @Override
         Builder printText(String text) {
-            return addLine("echo \"$text\"")
+            return addLine("echo $text")
```

Comment:
❓ AFAICT the behavior will be different when `text` includes newlines. Is this as intended?

Reply:
It wouldn't work with newlines either way. On Windows, the previous line was just printing out quotes. Looks like I never had a test to verify if the output is unquoted before.